package.json

```diff
@@ -31,24 +31,25 @@
     "version": "yarn build"
   },
   "dependencies": {
-    "@apollo/client": "^3.6.9",
+    "@apollo/client": "^3.7.3",
     "@formatjs/intl-relativetimeformat": "^4.5.9",
     "@formatjs/intl-utils": "^2.2.0",
     "@manuscripts/assets": "^0.6.1",
-    "@manuscripts/eslint-config": "^0.4.2",
-    "@manuscripts/title-editor": "^0.5.2",
+    "@manuscripts/title-editor": "../manuscripts-title-editor",
+    "@manuscripts/comment-editor": "../manuscripts-comment-editor",
+    "@manuscripts/manuscript-transform": "../manuscripts-manuscript-transform",
+    "@manuscripts/manuscripts-json-schema": "../manuscripts-json-schema",
```

Comment:
dependencies should be with a version!

Reply:
Check out my message in the group.

Reply:
I'v read the message but it's not clear to me how is that going to work during deployments?

Reply:
Or is that only for that specific change to be able to test things?

Reply:
This is only for local development, once these modules are released, this part will be updated with the new versions.